GG/GG/TextControl.h

```diff
@@ -248,6 +268,10 @@ class GG_API TextControl : public Control
     void RefreshCache();
     void PurgeCache();
 
+    /** Recompute line data, code points, text extent and minusable size cache when
+        m_text_elements changes.*/
```

Comment:
Meaning of "when m_text_elements changes". Does this immediately do something, or just set a flag so something happens later if something changes?